### ISSUE TITLE
Installation problems on Mac OS X Yosemite (Version 10.10.5)

### ISSUE THREAD

**Issue by bityogi** [bitcoin/bitcoin#7401]
This is probably a very low priority issue, but I still wanted to let you know about it.

When I try to run the 

`./configure --with-gui=qt5

The installation errors out, the final lines of the output are:

```
...
checking miniupnpc/upnperrors.h presence... yes
checking for miniupnpc/upnperrors.h... yes
checking for main in -lminiupnpc... (cached) yes
**configure: error: Qt dependencies not found**

```

When I run it without the --with-gui=qt5 option, it works fine.

I did install the dependencies as suggested in /doc/build-osx.md
`brew install autoconf automake berkeley-db4 libtool boost miniupnpc openssl pkg-config protobuf qt5

I did post this on [stack exchange](http://bitcoin.stackexchange.com/questions/42523/installation-troubles-compiling-from-source-code-on-a-mac-book-pro/42534#42534) as well.

**Comment by paveljanik** [comment]
Please post `config.log` file somewhere.

**Comment by jonasschnelli** [comment]
Try to uninstall "macports" (if you had it installed once), try to reinstall "homebrew".
Then call `brew install qt5` and try it again.

**Comment by bityogi** [comment]
@paveljanik : I've since installed it without the '--with=qt5' option and its been working fine. I wonder if this would have changed the config.log file, but regardless I am attaching it here. I did try to find the error 'qt dependencies not found' in it but couldn't locate them.

@jonasschnelli : You are probably right. I did have Bitcoin-Qt installed before from the downloads link on bitcoin.org website. But since everything is working now, I'm not sure I'll be able to replicate it.

Can I just close this ticket? If anyone else runs into this issue this comment thread I am sure will help them. 

[config.log.zip](https://github.com/bitcoin/bitcoin/files/102610/config.log.zip)
